### PR TITLE
Fix 59396

### LINKED ISSUE #59396: [BUG]SALT.STATES.WIN_PATH adds '.'(current directory) in path variable, windows 10
**Description**
Using the win_path adds '.' into the path variable

**Setup**
'C:/Users/{{client_username}}/AppData/Roaming/npm':
  win_path.exists

**Steps to Reproduce the behavior**
(Include debug logs if possible and relevant)

**Expected behavior**
Only add the name to path not an arbitrary '.' into path variable

**Versions Report**
<details><summary>salt --versions-report</summary>
(Provided by running salt --versions-report. Please also mention any differences in master/minion versions.) 

```
Salt Version:
          Salt: 3002.2
 
Dependency Versions:
          cffi: Not Installed
      cherrypy: Not Installed
      dateutil: 2.7.3
     docker-py: Not Installed
         gitdb: 2.0.6
     gitpython: 3.0.7
        Jinja2: 2.10.1
       libgit2: 0.28.3
      M2Crypto: Not Installed
          Mako: Not Installed
       msgpack: 0.6.2
  msgpack-pure: Not Installed
  mysql-python: Not Installed
     pycparser: Not Installed
      pycrypto: Not Installed
  pycryptodome: 3.6.1
        pygit2: 1.0.3
        Python: 3.8.5 (default, Jul 28 2020, 12:59:40)
  python-gnupg: 0.4.5
        PyYAML: 5.3.1
         PyZMQ: 18.1.1
         smmap: 2.0.5
       timelib: Not Installed
       Tornado: 4.5.3
           ZMQ: 4.3.2
```
</details>

**Additional context**

### PR DESCRIPTION
**Pull request by xeacott**
### What does this PR do?
Removes py2 that caused a period to be added to Path

### What issues does this PR fix or reference?
Fixes: https://github.com/saltstack/salt/issues/59396

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: salt/modules/win_path.py [hunk 1]
```diff
@@ -93,7 +86,7 @@ def get_path():
     ).split(";")
 
     # Trim ending backslash
-    return list(map(_normalize_dir, ret))
+    return ret
```
Comment (Ch3LL) [inline salt/modules/win_path.py:89]: Why are you removing normalizing the dirs? It looks like this helps to make comparisons with other normalized directories in the rest of the code. Also, there could be security considerations when removing normalizing the path.

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: salt/modules/win_path.py (hunk 1 above)
Comment (xeacott) [reply salt/modules/win_path.py:89]: Hmm, this change was the fix. But now after running it, the previous behavior is still there. Actually, testing with both versions of the return cause the issue, but only on subsequent runs and not the first run. That's really suspicious...

**Comment by Ch3LL** [comment]
Looks like there is a merge conflict. Once thats resolved is this ready for re-review?

**Comment by xeacott** [comment]
@Ch3LL should be good to go for re-review!

**Review by Ch3LL — APPROVED** [review]

**Comment by Ch3LL** [comment]
ping @twangboy can we get your review here as well?

**Review by twangboy — APPROVED** [review]

**Pull request merged**